src/Optimisers.jl

```diff
@@ -25,6 +26,8 @@ export Descent, Adam, Momentum, Nesterov, Rprop, RMSProp,
        WeightDecay, SignDecay, ClipGrad, ClipNorm, OptimiserChain, Lion,
        AccumGrad
 
+@compat public apply!, init, setup, update, update!
```

Comment:
```suggestion
VERSION >= v"1.11.0-DEV.469" && eval(Meta.parse("public apply!, init, setup, update, update!"))
```
Per https://discourse.julialang.org/t/is-compat-jl-worth-it-for-the-public-keyword/119041/2, we can avoid a dependency.